.eslintrc.js

```diff
@@ -39,6 +39,7 @@ module.exports = {
             },
         ],
         'no-unused-vars': ['error', { ignoreRestSiblings: true }],
+        '@typescript-eslint/no-unused-vars': ['error', { ignoreRestSiblings: true }],
```

Comment:
Hm I had initially thought that the first didn't apply to typescript files, only js, but tested it out again and I think that was wrong. The initial reasoning for adding a rule was because it was awkward to have `// eslint-disable-next-line @typescript-eslint/no-unused-vars` everywhere in our code, but adding `@typescript-eslint/no-unused-vars` doesn't seem to fix that issue. I don't think it makes sense to turn off warnings as well so I'm just going to revert this change.